Tiltfile

```diff
@@ -82,15 +82,15 @@ if not is_ci:
     )
 
 local_resource(
-    'build-plugin-backend',
+    'build-oncall-plugin-backend',
     labels=["OnCallPluginBackend"],
     dir="./grafana-plugin",
     cmd="mage buildAll",
     deps=['grafana-plugin/pkg/plugin']
 )
 
 local_resource(
-    'restart-plugin-backend',
+    'restart-oncall-plugin-backend',
```

Comment:
those are to avoid naming conflict between OnCall and Gops-Labels when you use ops-devenv